test/e2e/network/ingress.go

```diff
@@ -647,6 +647,38 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			}
 			verifyKubemciStatusHas(name, "is spread across 0 cluster")
 		})
+
+		It("single and multi-cluster ingresses should be able to exist together", func() {
```

Comment:
This looks great. Is it also worth trying things in the other order:
Create, Create, Delete Multi, Delete Single
?